client/src/pages/ProfileList/profileList.tsx

```diff
@@ -0,0 +1,80 @@
+import { LocalizationProvider, DesktopDatePicker } from '@mui/lab';
+import { Button, CircularProgress, InputLabel, TextField } from '@mui/material';
+import { Box } from '@mui/system';
+import React, { useState } from 'react';
+import InputAdornment from '@mui/material/InputAdornment';
+import SearchIcon from '@mui/icons-material/Search';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import useStyle from './useStyles';
+import Avatar from '@mui/material/Avatar';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea, Rating, Grid } from '@mui/material';
+import DateRangePicker, { DateRange } from '@mui/lab/DateRangePicker';
+import ProfileCard from './profileCard/ProfileCard';
+function ProfileList(): JSX.Element {
+  const classes = useStyle();
+  const cards = [1, 2, 3, 4, 5, 6];
+
+  return (
+    <>
+      <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
I think we can componentize the Search Bar for increased code maintainability

Reply:
created a new component SearchBar